static/js/settings_notifications.js

```diff
@@ -64,6 +67,30 @@ export function set_enable_digest_emails_visibility() {
     }
 }
 
+function setting_enabled(setting_name) {
```

Comment:
Maybe better to call this `warn_user_id_setting_disabled` -- the current name could be confusing.